packages/api-docs-builder/utils/parseSlotsAndClasses.ts

```diff
@@ -223,6 +224,7 @@ function extractSlots(
       description: getSymbolDescription(propertySymbol, project),
       default: tags.default?.text?.[0].text,
       class: slotClassDefinition?.className ?? null,
+      tutorial: tags.tutorial?.text?.[0].text,
```

Comment:
This line of code repeating for default, tutorial but they have common object properties

Can we create a function where we will pass corresponding as argument and it will return the first text property (text?.[0].text) ?

For example 
default: getTextFromTag(tags.default)
tutorial: getTextFromTag(tags.tutorial)

const getTextFromTag = (tag) => tag?.text?.[0]?.text || null;